File: MPI/mpicheck/tests/functions/1.0/MPI_Reduce_scatter.c
#include <mpi.h>
int main(char argc, char**argv)
{
    /* vars */
    const void *var_0;
    void *var_1;
    const int var_2[2];
    MPI_Datatype var_3;
    MPI_Op var_4;
    MPI_Comm var_5;
    int ret;
    /* calls */
    ret = MPI_Reduce_scatter(var_0, var_1, var_2, var_3, var_4, var_5);
    ret = PMPI_Reduce_scatter(var_0, var_1, var_2, var_3, var_4, var_5);
    return 0;
}

File: MPI/mpicheck/tests/functions/1.0/MPI_Reduce_scatter.f90

        
        program main
        use mpi
        
        TYPE(INTEGER) var_0(10)
       TYPE(INTEGER) var_1(10)
       INTEGER var_2(10)
       INTEGER var_3
       INTEGER var_4
       INTEGER var_5
       INTEGER var_6
        call mpi_reduce_scatter(var_0, var_1, var_2, var_3, var_4, var_5, var_6)
       call pmpi_reduce_scatter(var_0, var_1, var_2, var_3, var_4, var_5, var_6)
        end program main
    